bluetooth-toy-bb8/index.html

```diff
@@ -264,7 +264,7 @@
           if (controlCharacteristic == null) {
             navigator.bluetooth.requestDevice({
```

Comment:
Just got time to look at this. Agree with @jyasskin. The name prefix is too generic. Not accepting this pull request.